### ISSUE TITLE
Separate functions for loading data in Java and R

### ISSUE THREAD

**Issue by hdbeukel** [corehunter/corehunter3-r#21]
When reading data from a file it is currently loaded in Java and in R. For very large datasets this duplication may cause memory issues. Therefore it would be nice to have an option to not load the data in R but in Java only.

**Comment by daveneti** [comment]
If you use the database services and corehunter services, its all handled in Java. So I think this is best option. We can discuss.

**Comment by hdbeukel** [comment]
Well, the reason why I also load the data in R by default is because I figured this would be useful to for example explore the selected core in more detail. It could easily be omitted but I guess that's only necessary when running into trouble with memory right?

**Comment by daveneti** [comment]
Yes, I only see the need to use the services and/or alternative data structures if running into memory issues. I suggest you continue to use the simple implementations. Once version 3.1 is released, I can focus on this issue.

**Comment by hdbeukel** [comment]
This issue is not really related to the data structures. The question asked is whether the data should be loaded in Java only to select the core or in R as well to allow further analysis of the core. It's more a use case question than a data structure question. What do you think about this?

**Comment by daveneti** [comment]
I agree

**Comment by daveneti** [comment]
If R can handle it then read it in R as well. It should be an option to load only in java in the case there is a memory issue.

**Comment by hdbeukel** [comment]
And what about loading only in R? Seems useful as well, for example when you have previously constructed cores and wrote the selected IDs to a file and now just want to analyze these cores in R.

**Comment by daveneti** [comment]
Yes, that also should be an option.

**Comment by hdbeukel** [comment]
I guess it might be best to just split into two functions, one for loading in Java and one for R. This would avoid confusing the users.

This change requires quite some coding (especially reading in R without going through Java which currently handles the nasty details) so I suggest to postpone to version 3.2.

Agreed?

**Comment by hdbeukel** [comment]
When splitting the read functions we also need two write functions. See #22.

**Comment by hdbeukel** [comment]
Might no longer be vital due to the largely reduced memory footprint of the simple data implementations (see https://github.com/corehunter/corehunter3/issues/72).
